pyproject.toml

```diff
@@ -37,9 +37,9 @@ dependencies = [
     "psutil <= 7.0.0",
     "pybids <= 0.19.0",
     "requests",
-    "sdcflows <= 2.13.0",
+    "sdcflows ~= 2.13.0",
     "sentry-sdk <= 2.28.0",
-    "smriprep <= 0.18.0",
+    "smriprep ~= 0.18.0",
```

Comment:
The `~=` reflects that we try quite hard to maintain backwards compatibility during patch releases.